roles/integrations/tasks/main.yml

```diff
@@ -0,0 +1,63 @@
+---
```

Comment:
Might a bit easier to understand what is about if we name it `roles/db_integrations/` instead?